pkg/rancher-desktop/main/diagnostics/diagnostics.ts

```diff
@@ -72,6 +72,16 @@ export class DiagnosticsManager {
         await this.runChecker(checker);
       }
     });
+
```

Comment:
This is essentially repeated logic from the listener above. The only difference here is that we return our results after checking.

I'm considering if it makes sense to move the logic out into for the two listeners out to a separate function.

Reply:
Since it doesn't work anyway, it's probably better to just get rid of the `mainEvents.on('diagnostics-trigger')` and only keep this one. This means that even in places we don't care about the result we'd need to `/* (void) */ mainEvents.invoke('diagnostics-trigger')` (dropping the resulting `Promise<DiagnosticsCheckerResult>` on the floor) instead.